wpeverest/everest-forms
# Evf 497 Feature - Cloudfare Turnstile integration

### All Submissions:






### Changes proposed in this Pull Request:



Previously, We do not have Cloudflare Turnstile Captcha Integration. This PR will introduce Cloudflare Turnstile Captcha features.

### How to test the changes in this Pull Request:

1. Go to Settings > Captcha then choose Cloudflare Turnstile.
2.  Add Site and Secret Keys then choose theme mode.
3. then save the settings and verify whether it is working properly or not.

Note: For Testing Keys please Refer to the documentation https://developers.cloudflare.com/turnstile/reference/testing/ or create an account for site and secret keys.

### Types of changes:




### Other information:




### Changelog entry

> Evf 497 Feature - Cloudfare Turnstile integration.
